docs/Report.md

```diff
@@ -151,12 +151,21 @@ Before putting anything into the workflow actions, we create test manually to ru
 This can be found in the infrastructure tests in the tests for Cheep Repository. 
 We start by testing that what we want it to will work, and then we challenge it, by giving it some input that should throw validation exceptions. When we know both of these will pass, we can then move onto the workflows. 
 
-## Team work
-Show a screenshot of your project board right before hand-in. Briefly describe which tasks are still unresolved, i.e., which features are missing from your applications or which functionality is incomplete.
+# Teamwork
+This section will describe what features and implementation weren't completed and how the group worked with creation of issues and development. <br>
 
-Briefly describe and illustrate the flow of activities that happen from the new creation of an issue (task description), over development, etc. until a feature is finally merged into the main branch of your repository.
+## Project Board
+![Fix.XX Project Board](Images/ProjectBoard.png){width=60%}
+<br>
+This figure shows the Project board of Chirp at the day of hand-in. We have four issues that haven't been implemented before the deadline. All the four issues are under the Todo section but none of them are in the project requirement. That is to say they were formulated under the *Wild Style* development section. They were, adding tags to cheeps, being able to cheep a meme, trending cheeps and a re-cheep feature. As can be seen from the project board the re-cheep feature had an assigned developer but wasn't completed in time as other requirements had to be met. One Wild style feature was implemented, a like button for the cheeps. Although the like implementation is missing some functionality. A user can't see which cheep they've like and they can like a cheep infinitely many times.
+
```

Comment:
This figure shows the Project board of Chirp on the day of the hand-in. We have four issues that haven't been implemented before the deadline. All four issues are under the Todo section. None of them are in the project requirement. That is to say, they were formulated under the *Wildstyle* development section. They were, adding tags to cheeps, being able to cheep a meme, trending cheeps and a re-cheep feature. As can be seen from the project board the re-cheep feature had an assigned developer but wasn't completed in time as other requirements had to be met. One Wildstyle feature was implemented a like button on the Cheeps. Although the like implementation is missing some functionality. A user can't see which cheep they've liked and they can like, a Cheep infinitely many times.

Reply:
Thanks